src/Pyz/Yves/PaymentPage/Theme/default/templates/page-layout-payment/page-layout-payment.twig

```diff
@@ -0,0 +1,14 @@
+{% extends template('page-layout-payment', '@SprykerShop:PaymentPage') %}
+
+{% block content %}
+    <div class="box spacing-bottom--biggest">
```

Comment:
Again, it looks like we overriding the whole block just to add 1 class. Suggest to fix it on the core level so it's possible to extend it.